MainDemo.Wpf/Domain/PasswordHelper.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace MaterialDesignDemo.Domain;
+
+/// <summary>
+/// Simple helper class to enable data-binding for <see cref="PasswordBox.Password"/> which is not a <see cref="DependencyProperty"/>
+/// </summary>
+internal static class PasswordHelper
```

Comment:
I looked at this after writing up my comment on #2812.  I think for this class we should end up making it public (and moved into the main project) and renamed to `PasswordBoxAssist` (for naming consistency) to support that use case down the road.

Reply:
I agree. I'll try to work on a PR for doing the "reveal PasswordBox" when I have the time for it. Once that is in place, it could definitely replace this class here.